@here/harp-mapview/lib/MapView.ts

```diff
@@ -438,18 +438,30 @@ export interface MapViewOptions {
     numSecondChanceLabels?: number;
 
     /**
-     * The maximum distance for [[TextElement]] and icons, expressed as a fraction of the distance
+     * The maximum distance for [[TextElement]] to be rendered, expressed as a fraction of
+     * the distance between the near and far plane [0, 1.0].
+     * @default `0.99`.
+     */
+    maxDistanceRatioForTextLabels?: number;
+
+    /**
+     * The maximum distance for [[TextElement]] with icons, expressed as a fraction of the distance
      * between the near and far plane [0, 1.0].
      * @default `0.99`.
      */
-    maxDistanceRatioForLabels?: number;
+    maxDistanceRatioForPoiLabels?: number;
 
     /**
-     * The distance at which [[TextElement]]s start to apply their `distanceScale` value, expressed
-     * as a fraction of the distance between the near and far plane [0, 1.0].
-     * @default `0.4`.
+     * The minimum scaling factor that may be applied to [[TextElement]]s due to their distance.
+     * @default `0.7`.
      */
-    labelStartScaleDistance?: number;
+    labelDistanceScaleMin?: number;
+
+    /**
+     * The maximum scaling factor that may be applied to [[TextElement]]s due to their distance.
+     * @default `1.5`.
```

Comment:
Where does this ``1.5`` value come from? I don't see why you'd like to have text displayed bigger than designers intended (smaller makes sense for simulating perspective).

Reply:
It is the constraint for the closest labels visible ussualy if you set some very big tilt. Value comes from the native harp limits used.